_docs/tooling/maven.md

```diff
@@ -91,5 +92,20 @@ Here is the list of properties supported by the goal:
     increase compilation time, sometimes can make code even slower; 
     recommended for WebAssembly.
 * **targetType** &ndash; what code to generate. Following options are supported:
-  * *JAVASCRIPT*
-  * *WEBASSEMBLY*
\ No newline at end of file
+  * *JAVASCRIPT* to generate JavaScript
+  * *WEBASSEMBLY* to generate WebAssembly.
+  
+  The default value is JAVASCRIPT.
+* **stopOnErrors** The default value is `true`.
```

Comment:
Explanation?

Reply:
I do not know what it does. I did not mean to fix this documentation by this pull request. I merely wanted to be better. At least if these options are listed then later someone fixing the documentation has an easier job.